cli/tests/activate.bats

```diff
@@ -2857,7 +2868,7 @@ attach_runs_profile_twice() {
   # Our tcsh quoting appears to be broken so don't quote $TEARDOWN_FIFO
   FLOX_SHELL="$shell" "$FLOX_BIN" activate -- bash -c "echo > activate_finished && echo > $TEARDOWN_FIFO" >> output 2>&1 &
 
-  wait_for_background_activation
+  cat activated_finished
```

Comment:
Typo causing new failures (there are some catalog server failures too that I'm hoping will go away on the re-run):
```suggestion
  cat activate_finished
```

Reply:
Whoops fixed